src/components/auth/dataUpload.test.js

```diff
@@ -36,18 +36,4 @@ it("renders DataUpload component", () => {
   );
   let component = getAllByTestId(container, "DataUploadPage-test");
   expect(component.length).toBe(1);
-
-  let button = getAllByTestId(container, "clickLetterBoxd");
-  expect(button.length).toBe(1);
-
-  fireEvent.click(getByTestId(container, "clickLetterBoxd"));
-  component = getAllByText(container, "1.Log in to Letterboxd");
-  expect(component.length).toBe(1);
-
-  button = getAllByTestId(container, "clickIMDb");
-  expect(button.length).toBe(1);
-
-  fireEvent.click(getByTestId(container, "clickIMDb"));
-  component = getAllByText(container, "1.Log in to IMDb");
-  expect(component.length).toBe(1);
 });
```

Comment:
These are more strict implementations tests that can break easily depending on some UI changes like the fact that we are not rendering the IMDb portions as they are not functional.